### ISSUE TITLE
Regression: "q" to close broken in v5.0.4

### ISSUE THREAD

**Issue by sergiimk** [arijit79/minus#78]
**Describe the bug**
When upgrading from `5.0.3` to `5.0.4` I see that pressing "q" no longer closes the paginator.

Also something that broke pre `5.0.3` - when pager quits it no longer clears the output.

**To Reproduce**
My code is along the lines of:
```rust
let mut pager = minus::Pager::new();
pager.set_exit_strategy(minus::ExitStrategy::PagerQuit).unwrap();
...
minus::page_all(pager).unwrap();
```

**Expected behavior**
"q" closes the pager and clears the displayed output (same behavior as `git log` pagination).

**Desktop:**
 - OS: `Arch Linux 5.18.12-3-MANJARO`
- Terminal: `gnome-terminal`
- Shell: `zsh`
- `minus` Version: `5.0.4`

**Comment by arijit79** [comment]
I have pushed #79 containing fixes for this bug. Can you verify on your side whether everything is correct?

**Comment by sergiimk** [comment]
Thank you for the quick turnaround @arijit79. I have tried building `minus` from your branch and can confirm that:
- "q" now correctly closes the pager
- output is cleared upon exiting as well!

I also noticed what might be another regression - I can no longer select text when paging. Let me know if you'd like me to submit this as a separate issue.

Thanks, and looking forward to new release!

**Comment by arijit79** [comment]
> I also noticed what might be another regression - I can no longer select text when paging.

This is a problem with how crossterm works. When `crossterm::event::EnableMouseCapture` is executed, it captures all mouse actions including selection events. 

To fix this, we need to get all these selection events and keep a record of selected text accordingly.

I really want to fix this but unfortunately as you can see from the project activity I am not actively implementing new features because I am not getting much time to work on. If you want to help, you are more than welcome to file a PR. Join the Matrix or Discord room if you need any help. 

Thanks for filling the issue. I am closing this here.

**Comment by arijit79** [comment]
I have released v5.0.5 of minus containing fixes for this issue.
